ROMFS/px4fmu_common/init.d/airframes/17002_TF-AutoG2

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+#
+# @name TF:AutoG2
```

Comment:
```suggestion
# @name  ThunderFly:AutoG2
```
?

Reply:
Yes, this change is acceptable from us.

Reply:
Renamed

https://github.com/PX4/Firmware/blob/76b3ff04598f5a4a4a4e1487a93386d718cf4643/ROMFS/px4fmu_common/init.d/airframes/17002_TF-AutoG2#L3